src/main/app/claims/models/directions-questionnaire/directionsQuestionnaire.ts

```diff
@@ -67,6 +56,40 @@ export namespace DirectionsQuestionnaire {
     }
   }
 
+
+  export function toHearingLocation(directionsQuestionnaire: DirectionsQuestionnaireDraft): HearingLocation {
+
+    if(directionsQuestionnaire.hearingLocation === undefined || (directionsQuestionnaire.hearingLocation.courtName === undefined &&
+    directionsQuestionnaire.hearingLocation.alternativeCourtName === undefined)){
+      return undefined
+    }
+    return {
+      courtName: directionsQuestionnaire.hearingLocation &&
+      directionsQuestionnaire.hearingLocation.courtAccepted &&
+      directionsQuestionnaire.hearingLocation.courtAccepted.option === YesNoOption.YES ?
+        directionsQuestionnaire.hearingLocation.courtName : directionsQuestionnaire.hearingLocation.alternativeCourtName,
+      hearingLocationSlug: (directionsQuestionnaire.hearingLocationSlug && directionsQuestionnaire.hearingLocationSlug.length) ? directionsQuestionnaire.hearingLocationSlug : undefined,
+      courtAddress: undefined,
+      locationOption: toLocationOption(directionsQuestionnaire.hearingLocation),
+      exceptionalCircumstancesReason: directionsQuestionnaire.exceptionalCircumstances ?
+        directionsQuestionnaire.exceptionalCircumstances.reason : undefined
+    }
+  }
+
+  export function toLocationOption(hearingLocation: DraftHearingLocation): CourtLocationType {
```

Comment:
maybe we don't need to export function, i don't think we can or will use it anywhere else...

Reply:
done